bin/plugin/commands/performance.js

```diff
@@ -122,12 +122,14 @@ async function getPerformanceResultsForBranch(
 	environmentDirectory,
 	branch
 ) {
+	await git.discardLocalChanges( environmentDirectory );
+
 	log( '>> Fetching the ' + formats.success( branch ) + ' branch' );
 	await git.checkoutRemoteBranch( environmentDirectory, branch );
 
 	log( '>> Building the ' + formats.success( branch ) + ' branch' );
 	await runShellScript(
-		'npm install && npm run build',
+		'rm -rf node_modules && npm install && npm run build',
```

Comment:
Switching branches and npm install sometimes cause issues unless you remove the node modules. This is a bit less performant but more stable.

Reply:
> Switching branches and npm install sometimes cause issues unless you remove the node modules. This is a bit less performant but more stable.

Do you think there could be a similar issue with lingering build artifacts? The ones that would be removed by `npm run clean:packages` and `npm run clean:package-types`.

Reply:
I  thought `npm  run build` would take care of that?

Reply:
It seems the package cleaning is indeed performed but not the types one. I think it's not impactful for the performance tests but we maybe want to run the type cleaning as a "pre" command similar to the build cleaning.

Reply:
Mmm, yeah maybe. Worth a try